packages/nestjs-mongodb/README.md

```diff
@@ -0,0 +1,56 @@
+# NestJS MongoDB
+
+MongoDB, but for NestJS
+
+## Installation
+
+```sh
+yarn add @voiceflow/nestjs-mongodb mongodb
+yarn add -D @types/mongodb
+```
+
+## Usage
+
+The mongo module can be setup in a couple different ways using `forRootAsync`:
+
+- A `MongoOptions` object can be provided via `useValue`.
+- A `useFactory` function can be provided to return a `MongoOptions` object (or a promise for one!).
+- A class implementing `MongoOptions` can be provided using `useClass`.
+
+```ts
+import { MongoModule, MongoService, MongoOptions } from '@voiceflow/nestjs-mongodb';
+
+@Module({
+  imports: [
+    MongoModule.forRootAsync({
+      imports: [],
+
+      // Union field, one of `useValue`, `useFactory`, or `useClass`:
+      useValue: {
+        host: '0.0.0.0',
+        port: 6379,
```

Comment:
this should be updated to use the mongodb-specific options. maybe just change the port to `27017`? (`6379` is for redis)

Reply:
good call, I made it so it is the actual url :)